dev/com.ibm.ws.microprofile.health.2.0/resources/com/ibm/ws/microprofile/health20/resources/Health20.nlsprops

```diff
@@ -48,3 +48,8 @@ healthcheck.bean.call.exception.CWMH0050E.useraction=Review your call method wit
 healthcheck.application.down.CWMH0052W=CWMH0052W: The {0} implementing HealthCheckResponse in the {1} application in module {2}, reported a {3} status with data {4}.
 healthcheck.application.down.CWMH0052W.explanation=The DOWN status might cause a monitoring service to take action on this server.
 healthcheck.application.down.CWMH0052W.useraction=No user action is required.  This warning is a record of the DOWN status.
+
+# The Readiness HealthCheck had a DOWN outcome, when applications are not started yet
+readiness.healthcheck.applications.not.started.down.CWMH0053W=CWMH0053W: The readiness health check reported a DOWN overall status, because the following applications has not started yet: {0}
+readiness.healthcheck.applications.not.started.down.CWMH0053W.explanation=The specified applications have not completely started yet and are not ready to process requests, hence the readiness health check reported a DOWN status.
```

Comment:
Change
'readiness.healthcheck.applications.not.started.down.CWMH0053W.explanation=The specified applications have not completely started yet and are not ready to process requests, hence the readiness health check reported a DOWN status.'
to
'readiness.healthcheck.applications.not.started.down.CWMH0053W.explanation=The specified applications didn't start and are not ready to process requests, hence the readiness health check reported a DOWN status.'

Reply:
@ManasiGandhi When an app is started, it will take a little time changing its states.
[started.......initialization......ready].  This message describes the middle state where an app has been started but not ready for workload.